servicemanager/status.go

```diff
@@ -135,8 +135,11 @@ func (sm *ServiceManager) findStatuses() []serviceStatus {
 			if state.Started.Before(bootTime) {
 				// clean up state file
 				installDir, err := sm.findInstallDirOfService(state.Service)
-				if err != nil {
-					_ = sm.Ledger.ClearStateFile(installDir)
+				if err == nil {
+					err = sm.Ledger.ClearStateFile(installDir)
+					if err != nil {
+						panic(err)
```

Comment:
we probably don't want to panic outside of testing/debugging. could just print an error or something instead